### PR TITLE
Fix #67 - Support for nsFrom and nsTo options

### LINKED ISSUE #67: support nsFrom, nsTo
This given example restores a given dump (collection) made from `mydb` database namepsace into another namespace (`bibis` database): 
```
mongorestore  --host 127.0.0.1 --port 17017 --username root --password mypass --authenticationDatabase admin --nsInclude='mydb.*' --nsFrom 'mydb.*' --nsTo 'bibis.*' --archive=backup/mydb__2023-01-08_140815.gz --gzip
```

It would be great that MT Options supports `nsFrom` and `nsTo` options.

---
souces: question [SO](https://stackoverflow.com/questions/36321899/mongorestore-to-a-different-database) - mongodb tools [doc](https://www.mongodb.com/docs/database-tools/mongorestore/#change-collections--namespaces-during-restore)

### PR DESCRIPTION
**Pull request by mourtisma**
This fixes #67 .

It adds 2 optional parameters to the `restore` command:
- `dbFrom`: the source DB
- `dbTo`: the target DB
And their corresponding env vars.

If `db` is specified, then specifying those 2 parameter will not have any effect.

### CODE REVIEW

**Comment by mourtisma** [comment]
Hi @boly38 , this is a tentative contribution, for a feature that I think will soon be requested a lot.
I guess I will need you to approve a pending workflow.

Thanks a lot!

**Review by boly38 — DISMISSED** [review]
LGTM : thanks for this contribution

 - I am sincerely sorry for having left this post pending for so long. :/
 - I've done a mirror branch to test this PR in a workflow [first run](https://github.com/boly38/node-mongotools/actions/runs/8651377029/job/23721934694) / WiP

**Comment by boly38** [comment]
another UT failed : `should wrap restore commandConnectFromOptions uri ssl:` : on it

File: README.md [hunk 1]
```diff
@@ -248,8 +250,8 @@ You could create a dropbox app to get a token : cf. https://www.dropbox.com/deve
 
 | option             |  env                  |  required | default value   | description                                       |
 |--------------------|-----------------------|-----------|-----------------|---------------------------------------------------|
-| `dropboxToken`     | MT_DROPBOX_TOKEN      | false     | (none)          | activate dropbox feature if present               |
-| `dropboxLocalPath` | MT_DROPBOX_LOCAL_PATH | false     | "dropbox"       | local directory to receive dropbox dump           |
+| `dropboxToken`     |` MT_DROPBOX_TOKEN`      | false     | (none)          | activate dropbox feature if present               |
```
Comment (boly38) [inline README.md:253]: too much space + squash your fix @boly38 please

File: lib/MTWrapper.js [hunk 2]
```diff
@@ -44,14 +44,20 @@ class MTWrapper {
        }
        if ('db' in options && options.db != '*' && (!isRestore || isLegacyRestoreDb())) {// keep legacy capability
          command += ' --db ' + options.db;// deprecated for mongorestore: will produce additional deprecated stderr
-       } else if (isRestore && !isLegacyRestoreDb() && 'db' in options && options.db != '*') {
+       } else if (isRestore && !isLegacyRestoreDb() && 'db' in options && options.db != '*' && options.db != null) {
```
Comment (boly38) [inline lib/MTWrapper.js:47]: miss here too : re use tooling

**Review by boly38 — APPROVED** [review]
ok 👍

**Pull request merged**